src/pkg/retention/manager.go

```diff
@@ -16,17 +16,13 @@ package retention
 
 import (
 	"encoding/json"
-	"errors"
 	"fmt"
 	"time"
 
 	"github.com/astaxie/beego/orm"
-	cjob "github.com/goharbor/harbor/src/common/job"
-	"github.com/goharbor/harbor/src/jobservice/job"
 	"github.com/goharbor/harbor/src/pkg/retention/dao"
 	"github.com/goharbor/harbor/src/pkg/retention/dao/models"
 	"github.com/goharbor/harbor/src/pkg/retention/policy"
-	"github.com/goharbor/harbor/src/pkg/retention/q"
 )
 
 // Manager defines operations of managing policy
```

Comment:
the manager is for policy only here, can we move it to the ./retention/policy?

Reply:
it will introduce some "cycle import"